src/main/java/gov/cabinetofice/gapuserservice/web/LoginController.java

```diff
@@ -44,4 +47,15 @@ public RedirectView redirectOnLogin(@CookieValue(REDIRECT_URL_COOKIE) String red
     private RedirectView redirectToThirdParty() {
         return new RedirectView(authenticationProvider.getUrl()); //TODO make sure COLA's redirect URL points to the domain the user service sits on
     }
+
+    @GetMapping("/validate-user")
+    public ResponseEntity ValidateUser(final @CookieValue(name = USER_SERVICE_COOKIE_NAME, required = false ) String jwt) {
```

Comment:
Will admit I haven't been involved in any of the conversations around this method but in my mind we would want to return a boolean here rather than use HTTP response codes to suggest whether a user is logged in or not? 
401 suggests that the user has attempted to access an authenticated resource and does not possess the correct credentials, but this endpoint is not authenticated so not sure how suitable this is. 

Keen to hear your thoughts but I think we want to be returning `ResponseEntity.OK(isJwtValid)`